packages/rspack/src/config/schema.js

```diff
@@ -399,6 +399,13 @@ module.exports = {
 				css: {
 					description: "Enable native css support.",
 					type: "boolean"
+				},
+				rspackFuture: {
```

Comment:
Is this still necessary for config validation? @ahabhgk

Reply:
It's not. The validation is done by https://github.com/web-infra-dev/rspack/blob/main/packages/rspack/src/config/zod/experiments.ts.

Reply:
> It's not. The validation is done by https://github.com/web-infra-dev/rspack/blob/main/packages/rspack/src/config/zod/experiments.ts.

This is what exactly I'm thinking of. Let me remove this in the next one.